mitmproxy/tools/console/grideditor/editors.py

```diff
@@ -217,7 +217,7 @@ def __init__(
             ]) -> None:
```

Comment:
Let's also adjust the somewhat weird type signature here. This probably should be `typing.Any` instead of `str` in the last line if we intend to support ints.